Client/Frontend/Browser/Tabs/GroupedTabCell.swift

```diff
@@ -290,7 +290,8 @@ class GroupedTabContainerCell: UITableViewCell, UICollectionViewDelegateFlowLayo
         let cellWidth = floor((collectionView.bounds.width - GridTabTrayControllerUX.Margin * CGFloat(numberOfColumns + 1)) / CGFloat(numberOfColumns))
         let isIpad = UIDevice.current.userInterfaceIdiom == .pad
         let padding = isIpad && !UIWindow.isLandscape ? 75 : (isIpad && UIWindow.isLandscape) ? 105 : 10
-        return CGSize(width: Int(cellWidth) - padding, height: 185)
+        let width = (Int(cellWidth) - padding) >= 0 ? Int(cellWidth) - padding : 0
+        return CGSize(width: width, height: 185)
```

Comment:
This could be the root cause for crashes that are being seen by @isabelrios 🤞🏼

Reply:
I can't confirm :( I can't reproduce the crash on this branch or main. But let's give this a try 🤞 I still have my device on that status where it crashes in case you want to take a deeper look, even after landing this PR, don't feel blocked due to that.

Reply:
@nbhasin2 I was able to repro a crash in this PR, main and v38
<img width="340" alt="Captura de pantalla 2021-09-22 a las 10 49 48" src="https://user-images.githubusercontent.com/1897507/134315939-3d6c3d75-90f5-4064-8e12-a9df2b31a86a.png">
. 

Not sure if this is the same I was having, but just in case. I will send you the logs.